testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksWithHttpMetastore.java

```diff
@@ -0,0 +1,147 @@
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDatabricksWithHttpMetastore
+        extends BaseTestDeltaLakeS3Storage
+{
+    private String unityCatalogCatalogName;
+    private String unityCatalogExtLocation;
+
+    @BeforeMethodWithContext
+    @Override
+    public void setUp()
+    {
+        super.setUp();
+        unityCatalogCatalogName = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_NAME"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_NAME");
+        unityCatalogExtLocation = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_EXT_LOC"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_EXT_LOC");
+    }
+
+    @Test(groups = DELTA_LAKE_DATABRICKS_HTTP_HMS)
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testBasicTableReadWrite()
+    {
+        String schemaName = "testBasicConnection_" + randomNameSuffix();
+        String tableName = schemaName + ".testTable";
+        String tableLocation = createRandomTablePath(tableName);
+        try {
+            createSchemaInDelta(schemaName);
+            onDelta().executeQuery(format("CREATE TABLE %s.%s (c1 int, c2 string) using delta location 's3://%s/%s'", unityCatalogCatalogName, tableName, bucketName, tableLocation));
+            onDelta().executeQuery(format("INSERT INTO %s.%s values (1, 'one')", unityCatalogCatalogName, tableName));
+            // assert trino sees the schema
+            assertThat(onTrino().executeQuery("SHOW SCHEMAS FROM delta"))
+                    .contains(ImmutableList.of(
+                            row(schemaName.toLowerCase(Locale.ROOT))));
+            // assert trino sees the table
+            assertThat(onTrino().executeQuery("SHOW TABLES in delta." + schemaName))
+                    .containsOnly(row("testtable"));
+            // assert trino reads the right data from table
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s", tableName)))
+                    .containsOnly(row(1, "one"));
+            // assert trino can write into the table
+            onTrino().executeQuery(format("INSERT INTO delta.%s values (2, 'two')", tableName));
+            // assert both trino and databricks read the right data
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.%s order by c1", tableName)))
+                    .containsOnly(row(1, "one"), row(2, "two"));
+            assertThat(onDelta().executeQuery(format("SELECT * FROM %s.%s order by c1", unityCatalogCatalogName, tableName)))
+                    .containsOnly(row(1, "one"), row(2, "two"));
+        }
+        finally {
+            onDelta().executeQuery(format("DROP SCHEMA IF EXISTS %s.%s CASCADE", unityCatalogCatalogName, schemaName));
+        }
+    }
+
```

Comment:
Add a test showcasing that all DDL operations which are curently failing on Unity from Trino.

Reply:
> DDLs are currently not supported.

Please add a test reflecting this aspect.
Verify that while creating schemas or tables and exception is being thrown.

```
            assertThatThrownBy(() -> onTrino().executeQuery(CREATE TABLE ...."))
                    .hasMessageContaining("....");
```